components/EntryPreview/index.js

```diff
@@ -1,45 +1,148 @@
 import styled from "styled-components";
 import Link from "next/link";
+// Hooks
+import { useState } from "react";
 // SVGs
-import { ImageIcon } from "../svgs";
+import { ImageIcon, CaretRightIcon, CaretDownIcon } from "../svgs";
 
 const EntryPreview = ({ entryData }) => {
   const { _id: entryId } = entryData;
+  const [toggleExpander, setToggleExpander] = useState(false);
 
   return (
-    <StyledEntryPreviewCard href={`/${entryId}`}>
-      <StyledEntryName>{entryData.entryName}</StyledEntryName>
-      <StyledImageIcon />
-      <StyledTagContainer>
-        {entryData.entryTags.map((tag) => (
-          <StyledTagCard key={tag._id}>{tag.tagName}</StyledTagCard>
-        ))}
-      </StyledTagContainer>
-    </StyledEntryPreviewCard>
+    <>
+      <StyledEntryPreviewCard isexpanded={toggleExpander}>
```

Comment:
Just a note on naming: you should follow the naming conventions for props (camelCase):
```suggestion
      <StyledEntryPreviewCard isExpanded={toggleExpander}>
```

Reply:
Unfortunately react is not happy with that

Reply:
![Screenshot_7](https://github.com/dimitriosxmi/ArtistsReferenceOrganizer/assets/31593501/a299d876-1a7a-40d5-9b4b-cf07137a25dc)

Reply:
so that is unfortunately not an option I really wanted to, and also the `is-extended` is also not an option somehow... idk why but its weird.